mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -229,6 +229,13 @@ static cl::opt<std::string> tensorDataType("t", cl::desc("Data type for convolut
                                            cl::value_desc("Data type for convolution"),
                                            cl::init("f32"));
 
+static cl::opt<std::string> randomData(
+    "rand",
+    cl::desc(
+        "To specify the seed of random data generator for host validation"),
```

Comment:
@yiqian1 The description is still not clear to me.

This flag serves 2 purposes if I understand correctly:
- to use RNG or not
- the seed value

but the documentation doesn't say so.

Reply:
Also, `for host validation` is weird. One might interpret the data fed to GPU is not randomized?